app/serializers/books_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class BooksSerializer < ActiveModel::Serializer
```

Comment:
Serializer should be named `BookSerializer` instead of Book`s`Serializer. (also remember to rename file name from books_serializer.rb to book_serializer.rb)

Reply:
Done